Makefile

```diff
@@ -63,7 +63,8 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+	$(YARN) check
```

Comment:
I only spent a few moments reading about `yarn check`; it's possible I misunderstood its purpose. I believe it is run to confirm that __yarn.lock__ is consistent with __package.json__. If __package.json__ were to specify `foo@2.x.x` while __yarn.lock__ specified `foo@1.2.3`, `yarn lock` would exit with a non-zero exit code. This strikes me as a linting concern rather than a set-up concern. I would like to make this assertion each time I run `make lint` (assuming that it's not prohibitively slow). Let's move `$(YARN) check` from the `setup` target to the `lint` target.

If we're to require a contributor to update __yarn.lock__ if she updates a dependency in __package.json__, we shouldn't require her to spend five minutes reading the Yarn documentation to find out how to do so. Let's specify the build step in the makefile:

```make
yarn.lock:
	$(YARN) whatever-one-does-to-generate-lockfile
```